src/lambda_streams/Numbers.java

```diff
@@ -11,81 +11,248 @@ public static void main(String[] args) {
         System.out.println(nums);
 
         //Part II - refactor all of the class methods to accept lambda expressions. You can put the lambda functions directly inside the method calls, or defined them first, then pass them into the methods. give them the same names as the static methods, but add the word 'lambda' in front of every lambda function:
-        /* e.g.
+//
+        Calulate LambdaSumVar = (nums) -> {
+            int sum = 0;
+            for (int i = 0; i < nums.size(); i++) {
+                      sum = sum + nums.get(i);
+               }
+            System.out.println(sum);
+            return sum;
```

Comment:
Be sure to keep your variable names within the context of the code (e.g. a 'sum' variable is out of context for a 'subtraction' method).